CMakeLists.txt

```diff
@@ -425,6 +425,17 @@ if(WIN32)
   )
 endif()
 
+# Generate version header file
```

Comment:
Can you add a comment that this is done via CMake so that it uses version variables from the CMake environment which aren't directly available to our other code generation? That way it's clear why this particular file isn't generated via mako.